tests/tpch/generate_plot.py

```diff
@@ -16,7 +24,10 @@ def generate(outfile="chart.json", name=None, scale=None):
     df["library"] = df.path.map(lambda path: path.split("_")[-1].split(".")[0])
     df["query"] = df.name.map(lambda name: int(name.split("_")[-1]))
     df["name"] = df.cluster_name.map(lambda name: name.split("-", 3)[-1])
-    df["scale"] = df.cluster_name.map(lambda name: int(name.split("-")[2]))
+    try:
+        df["scale"] = df.cluster_name.map(lambda name: int(name.split("-")[2]))
+    except IndexError:
+        df["scale"] = 100  # default scale - cluster name wasn't formatted as expected
```

Comment:
This change doesn't make sense to me, and could result in misleading information.

Instead, I think that we should identify why the name wasn't formatted as expected.  If that's difficult to find out then escalate it to me, but adding this try-except isn't an appropriate solution I don't think.

Reply:
Sorry. That was not good indeed. 
I'm not sure how I ended up in such a scenario, and anyway can't get it to happen again locally so just removed it.